presidio-analyzer/tests/test_nlp_engine_provider.py

```diff
@@ -8,6 +8,7 @@
     StanzaNlpEngine,
     NlpEngineProvider,
 )
+from presidio_analyzer.nlp_engine.transformers_nlp_engine import TransformersNlpEngine
 
 
```

Comment:
What about E2E tests for the new engine?

Reply:
Totally agree. Current tests assume the package (transformers) is not installed, as this is a different flavor (pip install presidio-analyzer[transformers]. 
We should think about e2e in addition to unit tests that do install the package, potentially on a parallel pipeline. WDYT @shiranr @erann1987?

Reply:
sounds good. do you want to do it in this PR or another one?

Reply:
As this isn't part of the critical path and only adds new capability, I would suggest to finish the PR and do another solely for e2e tests

Reply:
There is also the docker integration to add before doing e2e tests